### ISSUE TITLE
Survivorship bias

### ISSUE THREAD

**Issue by neuhaus** [ciocan/cryptoroulette#1]
Do you only list coins that still exist today?
That would imply a strong survivorship bias in your data.

http://deadcoins.com/ is for real.

**Comment by ciocan** [comment]
I listed most traded top 250 with data from Coinmarketcap.
Most of them gained popularity last year after the ICO craze started last summer.

**Comment by neuhaus** [comment]
Yep, you should list the top 250 from last year, not the top 250 from today...
By only listing the top 250 of today you are only listing the winners.
